examples/.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
As in the tracer example: please use fusesoc from the source root, and put this gitignore there as well.